config/ldes/dispatch.ts

```diff
@@ -0,0 +1,184 @@
+import { moveTriples } from "../support";
+import { sparqlEscapeUri } from "mu";
+import { querySudo } from "@lblod/mu-auth-sudo";
+
+export default async function dispatch(changesets) {
+  // TODO: we could bundle the changesets with a sleep or something, sparql-parser can help here. 
```

Comment:
For now it might create a lot of entries in the feed, but I didn't want to complicate things. Also sparql parser is way more effective then mu-auth regarding sending changes out.